pkg/controllers/user/alert/watcher/event.go

```diff
@@ -0,0 +1,62 @@
+package watcher
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/rancher/rancher/pkg/controllers/user/alert/manager"
+	"github.com/rancher/types/apis/management.cattle.io/v3"
+	"github.com/rancher/types/config"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+type EventWatcher struct {
+	eventLister        v3.ClusterEventLister
```

Comment:
doesn't look like eventLister is being used, so should be safe to remove it from the list of fields.